plugin/trino-hudi/src/test/resources/hudi-testing-data/hudi_comprehensive_types_mor/README.md

```diff
@@ -0,0 +1,168 @@
+## Create script
+
+Structure of table:
+- MOR table with MDT enabled
+- Revision: 444cac26cb1077fd2b7deefc7b3713bacb270f9c
+
+```scala
+test("Create MOR table with comprehensive types") {
+    withTempDir { tmp =>
+        val tableName = "hudi_type_test_mor"
+        spark.sql(
+            s"""
+               |CREATE TABLE $tableName (
+               |  uuid STRING,
+               |  precombine_field LONG,
+               |
+               |  -- Numeric Types
+               |  col_boolean BOOLEAN,
+               |  col_tinyint TINYINT,
+               |  col_smallint SMALLINT,
+               |  col_int INT,
+               |  col_bigint BIGINT,
+               |  col_float FLOAT,
+               |  col_double DOUBLE,
+               |  col_decimal DECIMAL(10, 2),
```

Comment:
Parquet encodes the decimal values differently wrt precision and scale: https://github.com/apache/parquet-format/blob/master/LogicalTypes.md#decimal.  Should we test different precision and scales?

Reply:
TODO: bulk_insert to force row-writer path to check different decimal precisions and their underlying storage types. Then check if Trino can handle all different variations.